server/pull_request_handler.go

```diff
@@ -71,7 +78,9 @@ func (s *Server) pullRequestEventHandler(w http.ResponseWriter, r *http.Request)
 		s.setBlockStatusForPR(ctx, pr)
 	case "reopened":
 		mlog.Info("PR reopened", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number))
-		if err = s.handleCheckCLA(ctx, pr); err != nil {
+
+		_, err = s.handleCheckCLA(ctx, pr)
```

Comment:
Any reason to change this? We are not using `err` anywhere after.

Reply:
No. Reverting.